modules/go-compatibility-and-support-matrix.adoc

```diff
@@ -15,6 +15,7 @@ In the table, features are marked with the following statuses:
 |*OpenShift GitOps* 8+|*Component Versions*|*OpenShift Versions*
 
 |*Version* |*kam*    |*Helm*  |*Kustomize* |*Argo CD*|*ApplicationSet* |*Dex*     |*RH SSO* |*Notifications Controller*|
+|1.8.0    |0.0.47 TP |3.10.0 GA|4.5.7 GA   |2.6.1 GA |2.4.5 GA       |2.35.1 GA |7.5.1 GA |2.4.5 TP|4.8-4.12
```

Comment:
Question to SMEs: Review the support matrix versions and their support status for 1.8.0 and provide feedback.

- Refer to the confluence [GitOps Component Matrix](https://spaces.redhat.com/display/GITOPS/GitOps+Component+Matrix) page.
- The value for the ApplicationSet column is given as **n/a**. Please confirm the correct value that must go in the OCP docs?
- Also, I do not see the *Notifications Controller* column and its values on the confluence matric page. Do we need to add this information to the confluence page as well? Confirm this value and support status as well for v.1.8.0.release.